licensing/src/test/java/io/crate/license/CryptoUtilsTest.java

```diff
@@ -44,45 +40,34 @@
             data);
     }
 
-    private static void writeKeysToFiles(final KeyPair keyPair,
```

Comment:
I'm keeping this as an easy way to generate keys for testing. I'd prefer if we delete this later (before merging to master). 

What do you think?

But I guess I should have added a comment / todo for that, so thumps up

Reply:
Can you keep it in a branch? Or a script? The `license-management` branch should be clean